device/samples/pnp/readme.md

```diff
@@ -0,0 +1,37 @@
+# Samples to demonstrate Azure IoT Plug and Play
+
+The samples in this directory demonstrate how to implement an Azure IoT Plug and Play device.  Azure IoT Plug and Play is documented [here](aka.ms/iotpnp).  The samples assume basic familiarity with PnP concepts, though not in depth knowledge of the PnP "convention".  The "convention" is a set of rules for serializing and de-serialing data that uses IoTHub primitives for transport which the samples themselves implement.
```

Comment:
nit: change "PnP" to "Plug and Play" in all comments/text (two instances in 3rd sentence of the paragraph)